<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cp2kInputDialog</class>
 <widget class="QDialog" name="Cp2kInputDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CP2K Input</string>
  </property>
  <layout class="QGridLayout">
   <item row="3" column="0">
    <widget class="QTextEdit" name="previewText">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="resetAllButton">
       <property name="text">
        <string>Reset All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="defaultsButton">
       <property name="text">
        <string>Defaults</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="computeButton">
       <property name="text">
        <string>Submit Calculation...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="generateButton">
       <property name="text">
        <string>Save File...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QTabWidget" name="modeTab">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="basicWidget">
      <attribute name="title">
       <string>&amp;Basic Setup</string>
      </attribute>
      <layout class="QGridLayout">
       <item row="9" column="3">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QComboBox" name="chargeCombo"/>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="7" column="1">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>In:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>stateCombo</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Basis set:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>basisCombo</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <layout class="QHBoxLayout">
         <item>
          <widget class="QComboBox" name="calculateCombo"/>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="6" column="3">
        <layout class="QHBoxLayout">
         <item>
          <widget class="QComboBox" name="basisCombo"/>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Functional</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="functionalCombo">
           <property name="accessibleDescription">
            <string notr="true"/>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>16</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="9" column="1">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Charge:</string>
         </property>
         <property name="buddy">
          <cstring>chargeCombo</cstring>
         </property>
        </widget>
       </item>
       <item row="7" column="3">
        <layout class="QHBoxLayout">
         <item>
          <widget class="QComboBox" name="stateCombo"/>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="10" column="1">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Calculate:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>calculateCombo</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Filename Base:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Title:</string>
         </property>
         <property name="buddy">
          <cstring>titleEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Method:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>methodCombo</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLineEdit" name="titleEdit"/>
       </item>
       <item row="3" column="3">
        <widget class="QLineEdit" name="baseNameEdit">
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>job</string>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <layout class="QHBoxLayout">
         <item>
          <widget class="QComboBox" name="methodCombo"/>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>MM</string>
      </attribute>
      <widget class="QSplitter" name="splitter">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>541</width>
         <height>35</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Emax Spline</string>
        </property>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QSpinBox" name="emaxSplineSpin">
           <property name="maximum">
            <number>99999</number>
           </property>
           <property name="value">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>QM</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>QM/MM</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>calculateCombo</tabstop>
  <tabstop>basisCombo</tabstop>
  <tabstop>functionalCombo</tabstop>
  <tabstop>stateCombo</tabstop>
  <tabstop>methodCombo</tabstop>
  <tabstop>chargeCombo</tabstop>
  <tabstop>previewText</tabstop>
  <tabstop>resetAllButton</tabstop>
  <tabstop>defaultsButton</tabstop>
  <tabstop>generateButton</tabstop>
  <tabstop>closeButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
